### ISSUE TITLE
joinGroup throws a marshal exception. 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#65]
```
What steps will reproduce the problem?
1. Attempt to call joinGroup(groupId) from the API with a valid group ID


What is the expected output? What do you see instead?
Client successfully joins the group, or an LinkedIn related exception. Instead, 
a javax.xml.bind.MarshalException is thrown, 

What version of the product are you using? On what operating system?
-V1.0 according to maven, 1.0.410 from site.  

Please provide any additional information below.


```

Original issue reported on code.google.com by `dave.z...@yupiq.com` on 17 Aug 2011 at 6:57

**Comment by GoogleCodeExporter** [comment]
```
This issue is reproducible.
```

Original comment by `nabeelmukhtar` on 19 Aug 2011 at 8:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
It looks like the problem is simply that the JAXB annotation on the Membership 
object has fields which are marked required but can actually be null in the 
case of joining a group. 

Simply removing the required restriction on the annotations themselves seems to 
have solved the problem. 
```

Original comment by `mrclea...@gmail.com` on 19 Aug 2011 at 8:16

**Comment by GoogleCodeExporter** [comment]
```
This issue is fixed in the release 1.0.415.
```

Original comment by `nabeelmukhtar` on 5 Sep 2011 at 12:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hi All,
I am getting the same problem however i am using the latest jar file release 
1.0.429.
while executing client.joinGroup(group_id).please help me asap
thanks
```

Original comment by `akkygr...@gmail.com` on 31 Jul 2012 at 12:12
